### PR TITLE
[TASK] Drop support for Rails 3.2 and 4.0

### LINKED ISSUE #25: Drop support for Rails 3.2 and 4.0 after Rails 5.0 has been released
http://guides.rubyonrails.org/maintenance_policy.html

### PR DESCRIPTION
**Pull request by oliverklee**
There will not be any updates anymore for Rails 3.2, 4.0 or 4.1
according to http://guides.rubyonrails.org/maintenance_policy.html .

Fixes #25

### CODE REVIEW

**Comment by lwe** [comment]
👍 will require making a breaking release, but works for me, thanks

File: page_title_helper.gemspec [hunk 1]
```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |s|
 
   s.license          = 'MIT'
 
-  s.add_dependency 'rails', '>= 3.2.0'
+  s.add_dependency 'rails', '>= 4.2.0'
```
Comment (lwe) [inline page_title_helper.gemspec]: maybe would make sense to also add `<= 6` as a requirement?

Comment (oliverklee) [inline page_title_helper.gemspec]: I've just added "< 6.0" and re-pushed. Thanks!

**Comment by lwe** [comment]
Thanks

**Pull request merged**